libs/langchain-openai/src/tests/chat_models.standard.int.test.ts

```diff
@@ -65,3 +111,26 @@ test("ChatOpenAIStandardIntegrationTests", async () => {
   const testResults = await testClass.runTests();
   expect(testResults).toBe(true);
 });
+
+async function invoke(
```

Comment:
I like this pattern! Might be useful to add to the base standard tests library as an exported util. Not important to add for now though